package.json

```diff
@@ -0,0 +1,60 @@
+{
+  "name": "rod",
+  "version": "1.0.0",
+  "main": "index.js",
+  "description": "Return Of Documnents (ROD)",
+  "engines": {
+    "node": ">=18.12.0"
+  },
+  "scripts": {
+    "start": "node server.js",
+    "start:dev": "hof-build watch --env",
+    "test": "NODE_ENV=test yarn run test:lint && yarn run test:unit",
+    "test:lint": "eslint . --config ./node_modules/eslint-config-hof/default.js",
+    "build": "hof-build",
+    "postinstall": "hof-build",
+    "test:unit": "nyc _mocha \"test/_unit/**/*.spec.js\"",
+    "test:snyk": "snyk config set api=SNYK_TOKEN && snyk test"
+  },
+  "dependencies": {
+    "accessible-autocomplete": "^2.0.4",
+    "axios": "^1.6.5",
+    "bl": "^6.0.10",
+    "busboy": "^1.6.0",
+    "bytes": "^3.1.2",
+    "dotenv": "^16.3.1",
+    "eslint-config-hof": "^1.3.1",
+    "hof": "~20.5.0",
+    "jquery": "^3.7.1",
+    "lodash": "^4.17.21",
+    "moment": "^2.30.1",
```

Comment:
You may be able to remove jquery and moment.

Reply:
done